src/main/webapp/main.jsp

```diff
@@ -14,6 +14,12 @@
     <script src="javascript/main.js"></script>
 </head>
 <body onload="displayActiveTrips()">
+<div  id="logout">
```

Comment:
I assume there is  a redundant space between 'div' and 'id'. Could you please remove it.